python/lsst/pipe/supertask/activator.py

```diff
@@ -131,6 +132,34 @@ def activate(self):
         """
         pass
 
+def get_inheritance(stask_class, subclasses=[]):
```

Comment:
This is going to bite. The Python idiom is to use `subclasses=None` here unless you want the same list to be used every single time.